app/src/main/java/fr/free/nrw/commons/mwapi/OkHttpJsonApiClient.java

```diff
@@ -328,6 +329,76 @@ public List<Place> getNearbyPlaces(final LatLng cur, final String language, fina
         throw new Exception(response.message());
     }
 
+    /**
+     * Retrieves nearby places based on screen coordinates and optional query parameters.
+     *
+     * @param screenTopRight       The top right corner of the screen (latitude, longitude).
+     * @param screenBottomLeft     The bottom left corner of the screen (latitude, longitude).
+     * @param language             The language for the query.
+     * @param shouldQueryForMonuments  Flag indicating whether to include monuments in the query.
+     * @param customQuery          Optional custom SPARQL query to use instead of default queries.
+     * @return                     A list of nearby places.
+     * @throws Exception           If an error occurs during the retrieval process.
+     */
+    @Nullable
+    public List<Place> getNearbyPlaces(
+        final fr.free.nrw.commons.location.LatLng screenTopRight,
+        final fr.free.nrw.commons.location.LatLng screenBottomLeft,final String language,
```

Comment:
please insert a space after each `,` comma, here and in the rest of the PR (many occurrences)